### ISSUE TITLE
pip install blockchain fails

### ISSUE THREAD

**Issue by polinom** [blockchain/api-v1-client-python#87]
When I run 
```
$ pip install blockchain
```
I get
```
Collecting blockchain
  Using cached https://files.pythonhosted.org/packages/2b/48/0f87faf9ad42a94fcbdcc850efaa56c3a0319f3afe9c256ed24d04120cb0/blockchain-1.4.2.tar.gz
Collecting enum (from blockchain)
  Using cached https://files.pythonhosted.org/packages/0c/4e/1ea357e7783c756bb579333c1e4a026fb331371ee771f616ffedc781e531/enum-0.4.6.tar.gz
    Complete output from command python setup.py egg_info:
    /Users/paul.szabo/igor/psr-env-2/lib/python3.6/site-packages/setuptools/version.py:1: UserWarning: Module enum was already imported from /Users/paul.szabo/.pyenv/versions/3.6.3/lib/python3.6/enum.py, but /private/var/folders/yt/dr_ygt812f1_rlkxpzv638ljv9q1bs/T/pip-build-jmw_khbp/enum is being added to sys.path
      import pkg_resources
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/yt/dr_ygt812f1_rlkxpzv638ljv9q1bs/T/pip-build-jmw_khbp/enum/setup.py", line 24, in <module>
        version = main_module.__version__
    AttributeError: module 'enum' has no attribute '__version__'
```

**Comment by polinom** [comment]
Seems like https://github.com/blockchain/api-v1-client-python/pull/86 addresses it?

**Comment by RayXY** [comment]
Issue fixed with 1.4.3 release
